wiki/Tournaments/JHC/JHC_2024/en.md

```diff
@@ -56,8 +60,264 @@ The **JHC 2024** (***Jack House Cup 2024***) is a team–based osu!mania 4K tour
+| 46 | Shangus | [lianic](https://osu.ppy.sh/users/13975661) |
+| 47 | soloist | [Shirobi](https://osu.ppy.sh/users/18804586) |
+| 48 | SV Player | [\[Crz\]yomogi237](https://osu.ppy.sh/users/28571440) |
+| 49 | Team White Dress | [lonelyxvi](https://osu.ppy.sh/users/24133264) |
+| 50 | Techno Music | [neeko the rock](https://osu.ppy.sh/users/13471840) |
+| 51 | THE TROLLERS | [Takedown05](https://osu.ppy.sh/users/18060303), [\[GE\] Arthur](https://osu.ppy.sh/users/24122055) |
+| 52 | THE TRUTH OF NOCENT | [ERA Veryi](https://osu.ppy.sh/users/13878539), [\[Crz\]Nickname](https://osu.ppy.sh/users/10817494) |
+| 53 | We can't late game | [SHOKO \-](https://osu.ppy.sh/users/10279743), [MatchaLatte\-](https://osu.ppy.sh/users/15944956) |
+| 54 | We can't mid game | [\- Rinmoz \-](https://osu.ppy.sh/users/16639144), [Freshky](https://osu.ppy.sh/users/11959687) |
+| 55 | wehehe | [Eliminate](https://osu.ppy.sh/users/9169747), [myucchii](https://osu.ppy.sh/users/10072733) |
+| 56 | ♿🍔♿🀄♿ | [\[GB\]Burger king](https://osu.ppy.sh/users/9841162), [MidRed](https://osu.ppy.sh/users/17641994) |
+| 57 | まひろ隊 | [tianzitanhua](https://osu.ppy.sh/users/32196635) |
+| 58 | 不喜欢打叠 | [GinaFujino](https://osu.ppy.sh/users/25864079), [XeOF4](https://osu.ppy.sh/users/22692196) |
+| 59 | 原核生物队 | [Infinityyyyyyyy](https://osu.ppy.sh/users/26233818) |
+| 60 | 取队伍名称好难 | [\[GB\]Lingyu](https://osu.ppy.sh/users/29743849), [Takoke](https://osu.ppy.sh/users/11628037) |
+| 61 | 大天空killers | [\[Crz\]Yurucawaii](https://osu.ppy.sh/users/25682415), [\[Crz\]bubu](https://osu.ppy.sh/users/28251667) |
+| 62 | 帝者战神白武男 | [\_Glass\_](https://osu.ppy.sh/users/21239735) |
+| 63 | 影流之主 | [\[GB\]QAQlingjiu](https://osu.ppy.sh/users/27288518) |
+| 64 | 快手极速版 | [Na2He](https://osu.ppy.sh/users/20298842), [Az-Q](https://osu.ppy.sh/users/32613475) |
+| 65 | 憋笑 | [chino3310](https://osu.ppy.sh/users/16626292), [\-Thirteen\-](https://osu.ppy.sh/users/34241263) |
+| 66 | 我先睡了队 | [WELUVMYUKA](https://osu.ppy.sh/users/29949157), [chana](https://osu.ppy.sh/users/18375016) |
+| 67 | 打胶队 | [qiaolilailai](https://osu.ppy.sh/users/13809991), [\-\-achang\-\-](https://osu.ppy.sh/users/31485150) |
+| 68 | 海陆 | [CaptainLan](https://osu.ppy.sh/users/18483399) |
+| 69 | 痛恨叠键 | [sarc6o9](https://osu.ppy.sh/users/23135250) |
+| 70 | 白 | [Lonann](https://osu.ppy.sh/users/14617751) |
+| 71 | 绝不认输 | [Styrene](https://osu.ppy.sh/users/16998777) |
+| 72 | 👍 | [TheOPmeme](https://osu.ppy.sh/users/15763622) |
+
+## Podium
+
+| Rank | Team |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "冠军") | wehehe |
+| ![Silver crown](/wiki/shared/crown-silver.png "亚军") | GangWonDuo |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "季军") | hsod |
+| \#4 | garfield exterminators if fortnite |
+| \#5 ~ \#6 | hawk |
+| \#5 ~ \#6 | SV Player |
+| \#7 ~ \#8 | Shangus |
+| \#7 ~ \#8 | 👍 |
+| \#9 ~ \#12 | ETA |
+| \#9 ~ \#12 | THE TRUTH OF NOCENT |
+| \#9 ~ \#12 | 原核生物队 |
+| \#9 ~ \#12 | 大天空killers |
```

Comment:
you have duplicate rows (same placements), please combine where appropriate

use `–` (en dash) instead of `~`. `\` is not necessary